### PR TITLE
fish-tank 0.1.1 (new formula)

### PR DESCRIPTION
**Pull request by bpinto**
New formula for fish-tank.

### CODE REVIEW

File: Library/Formula/fish-tank.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/joelthelion/autojump.git"
+
+  def install
+    system "make", "install"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Need to specify the prefix.

File: Library/Formula/fish-tank.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  test do
+    system "false"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Needs a test.

File: Library/Formula/fish-tank.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/joelthelion/autojump.git"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: This doesn't match the project above.

Comment (bpinto) [inline Library/Formula/fish-tank.rb]: Oops, wrong copy and paste.

File: Library/Formula/fish-tank.rb (hunk 1 above)
Comment (bpinto) [inline Library/Formula/fish-tank.rb]: What prefix? This is the repo documentation on how to install it.

```
git clone git://github.com/terlar/fish-tank.git
cd fish-tank
make install
```

File: Library/Formula/fish-tank.rb (hunk 2 above)
Comment (bpinto) [inline Library/Formula/fish-tank.rb]: I don't know any test to add to this formula. Any ideas?

File: Library/Formula/fish-tank.rb (hunk 1 above)
Comment (tdsmith) [inline Library/Formula/fish-tank.rb]: This formula will need to install to the prefix defined by `prefix`, which will be a path like `/wherever/homebrew/is/installed/Cellar/fish-tank/1.2.3`. Looking at the Makefile, `system "make", "install", "PREFIX=#{prefix}"` is probably the right incantation.

File: Library/Formula/fish-tank.rb (hunk 2 above)
Comment (tdsmith) [inline Library/Formula/fish-tank.rb]: If `fish-tank` runs tests in `fish`, you should have `fish-tank` run a test you expect to pass in the formula's test.

This formula should `depends_on "fish"` if it is not useful without fish.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/fish-tank.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/terlar/fish-tank.git"
+
+  def install
+    system "make", "install"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Need to specify a `prefix`.

File: Library/Formula/fish-tank.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Use `sha256`

File: Library/Formula/fish-tank.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Needs a `desc` (description)

File: Library/Formula/fish-tank.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/terlar/fish-tank.git"
+
+  def install
+    system "make", "install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/fish-tank.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by bpinto** [comment]
I'm not using this anymore, so I'm closing it. Thanks!

**Pull request closed without merging**